openapi/components/schemas/OrderItem.yaml

```
@@ -23,6 +23,61 @@ properties:
     anyOf:
       - $ref: ./OriginalPlan.yaml
       - $ref: ./FlexiblePlan.yaml
+  usageLimits:
+    type: object
+    nullable: true
+    description: |-
+      Set the soft and hard usage limits thresholds for an item with metered billing plan.
+      This value is ignored when the plan is not metered.
+    properties:
+      softLimit:
+        type: object
+        nullable: true
+        description: |-
+          Soft limit configures thresholds for notification purposes.
+          When the reported usage reaches 75%, 90% and 100% of the configured limit, a specific event is fired.
+          Webhook and a notification can be configured for this event to notify the customer.
+          Useful for accounting and customer success purposes.
+          Metered billing plan usages can still be reported if the soft limit is reached.
+        minProperties: 1
+        maxProperties: 1
+        properties:
+          quantity:
+            type: integer
+            description: Usage limit quantity.
+            nullable: true
+            minimum: 1
+          amount:
+            type: number
+            format: double
+            description: Usage limit amount in the order currency.
```

suggestion: for readability

on first read, I interpreted this as a limit on the order in which the currencies are ordered 

```suggestion
            description: Usage limit amount in the currency of the order.
```